pinot-spi/src/main/java/org/apache/pinot/spi/utils/retry/RetryPolicy.java

```diff
@@ -33,7 +33,8 @@ public interface RetryPolicy {
    * @param operation The operation to attempt, which returns true on success and false on failure.
    * @throws AttemptsExceededException
    * @throws RetriableOperationException
+   * @return the number of attempts used for the operation. 0 means the first try was successful.
```

Comment:
(nit) return the number of attempts used for the operation, upon successful attempt? In case of failure this wouldn't work right?

Reply:
If you are referring to the exception case, I do not think the programmer should expect any return value there.